platform/darwin/src/MGLLight.mm.ejs

```diff
@@ -111,4 +113,23 @@ NS_INLINE mbgl::style::TransitionOptions MGLOptionsFromTransition(MGLTransition
     return mbglLight;
 }
 
+<% if (properties.length) { -%>
+
+<% for (const property of properties) { -%>
+
+- (void)set<%- camelize(property.name) %>:(NSExpression *)<%- objCName(property) %> {
+    MGLLogDebug(@"Setting <%- objCName(property) %>: %@", <%- objCName(property) %>);
+
```

Comment:
Nit: the line breaks in these new setters are inconsistent — this includes a blank line between the log and assignment, while `transition` below does not.